src/docs/quick-start.rst

```diff
@@ -1,12 +1,182 @@
+
+.. code-block:: text
+ 
+    id admin
+
+If you have auto mounts configured, run ipa-client-automount to enable that feature. Use the -U flag for unattended.
+
+.. code-block:: text
+
+    ipa-client-automount -U
+
+
+Quick Start AD
+**************
+
+Before starting make sure you have the following information:
+
+- Domain user credentials *i.e. Administrator*
+- AD Domain name *i.e. sssd.io*
+- AD Server IP *i.e. 1.2.3.4*
+- AD Server hostname *i.e. ad1.sssd.io*
+
+Install the necessary packages, for RHEL and clones the packages are *sssd, adcli, realm, oddjob, and oddjob-mkhomedir*
+
+.. code-block:: text
+
+    yum install -y sssd adcli realm oddjob oddjob-mkhomedir
+
+Make sure DNS is pointing to your AD server(s) by checking **/etc/resolv.conf** and it having it contain an entry with an AD server IP, if it’s missing go ahead and add it, as the first entry.
+
+.. code-block:: text
+
+    search sssd.io
+    nameserver 1.2.3.4
+    nameserver 1.1.1.1
+   
+Now you can issue the realm join command with the domain name in order to join the domain.
+
+.. code-block:: text
+
+    realm join sssd.io 
+
+It will default and use the *Administrator* user, add the -u flag to specify a different user account to join the domain. 
+
+.. code-block:: text
+
+    realm join -u jsmith sssd.io 
+
+Now see if it works, and issue an id command.
+
+.. code-block:: text
+
+    id administrator@sssd.io
+    
+If you want to use short names, edit **sssd.conf** and set **use_fully_qualified_names** to **false**.
+
+.. note::
+
+    To avoid conflicts when using short names in a multidomain environment, configure SSSD using :doc:`short names <../design-pages/pages/shortnames>`.
+
+.. code-block:: text
+
+    id administrator
+
+The following command and logins should now work. For more detail please refer to :doc:`direct ad page <..direct-ad>`.
+
+Quick Start LDAP
+****************
+
+Before starting make sure you have the following information:
+
+- LDAP domain *i.e. sssd.io*
+- LDAP suffix *i.e. DC=sssd,DC=io*
+- LDAP bind user *i.e. UID=bind_user,OU=people,DC=sssd,DC=io*
+- LDAP bind password *i.e. password123*
+- LDAP server hostname *i.e. ldap1.sssd.io*
+- LDAP server IP *i.e. 1.2.3.4*
+- LDAP server CA certificate *i.e. /etc/openldap/cacerts/ca.crt*
+
+.. note::
+
+    The bind user and the bind password are only necessary if the
+    LDAP server you are connecting does not permit anonymous binds.
+
+First install the necessary package, sssd.
+
+.. code-block:: text
+
+    yum install -y sssd
+
+Edit */etc/sssd/sssd.conf* and add a new domain section. The section should look like the following without a bind user.
+
+.. code-block:: ini
+   :caption: /etc/sssd/sssd.conf
+
+    [domain/LDAP_DOMAIN]
+    id_provider = ldap
+    auth_provider = ldap
+
+    ldap_uri = ldap://ldap1.sssd.io
+    ldap_search_base = DC=sssd,DC=io
+
+    ldap_id_use_start_tls = true
+    ldap_tls_reqcert = demand
+    ldap_tls_cacert = /etc/openldap/cacerts/ca.crt
```

Comment:
Do you also need to include this? or will it be autodiscovered, I don't remember.
~~~
[sssd]
domains = LDAP_DOMAIN
~~~

Reply:
This is needed.